notecard/main.go

```diff
@@ -86,7 +88,14 @@ func setupNotecard(protocol string) (*notecard.Context, error) {
 	}
 
 	if protocol == Serial {
-		card, err := notecard.OpenSerial("/dev/tty.usbmodemNOTE1", 9600)
+		serialPort := getEnv("NOTECARD_SERIAL_DEVICE", SerialDevice)
+
+		baud, err := strconv.Atoi(getEnv("NOTECARD_SERIAL_BAUD", "9600"))
```

Comment:
why make a const for the default device, and not for the default baud rate ? ;P

Reply:
good point